app/uk/gov/hmrc/testuser/connectors/ApiPlatformTestUserConnector.scala

```diff
@@ -41,7 +44,8 @@ class ApiPlatformTestUserConnector @Inject() (
 
   private val bearerToken = servicesConfig.getConfString(s"$serviceKey.bearer-token", "")
 
-  private val httpClient = proxiedHttpClient.withAuthorization(bearerToken)
+  private def configureEbridgeIfRequired(requestBuilder: RequestBuilder): RequestBuilder =
```

Comment:
Do we need to check if a proxy is required, since previously it was always proxied? I don't think there's a complex ebridge setup, looking at application.conf, unlike third-party-orchestrator and api-platform-microservice.

Reply:
Fixed